<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HareetTashfer</class>
 <widget class="QMainWindow" name="HareetTashfer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HareetTashfer - Arabic to Code Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>HareetTashferLogo.png</normaloff>HareetTashferLogo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Copy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>انسخ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Convert">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>حول</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_29">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_30">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;خ&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ح&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ث&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ج&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ا&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ت&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ب&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;د&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ز&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ش&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;س&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ر&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ذ&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_12">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ص&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_13">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_18">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_15">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_19">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;غ&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_16">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_15">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ض&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_18">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ع&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_17">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_20">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ف&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ق&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_20">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_17">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ظ&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_16">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ط&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_24">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_28">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ي&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_27">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;و&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_26">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_25">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_28">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_25">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ن&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_27">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_26">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ه&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_22">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_23">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_24">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;م&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_22">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ك&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_23">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot;ل&quot;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="T_0">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="L_0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>تشفير &quot; &quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="LT_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>259</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ادخل العربي</string>
    </property>
   </widget>
   <widget class="QLabel" name="LT_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>بعد التحويل</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
